modules/configure-ocp-oauth-identity-provider.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-user-access/configure-ocp-oauth.adoc
+:_module-type: PROCEDURE
+[id="configure-ocp-oauth-identity-provider_{context}"]
+= Configuring {ocp} Oauth server as an identity provider in {product-title}
+
+[role="_abstract"]
+Use the instructions in this section to integrate the built-in {ocp} OAuth server as an identity provider for {product-title}.
+
+.Prerequisites
+* You must have permissions to configure identity providers in {product-title}.
+* You must have already configured users and groups in {ocp} OAuth server.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Access Control*.
+. Open the *Add an Auth Provider* menu and select *OpenShift Auth*.
+. Enter a name for the authentication provider in the *Name* field.
+. Choose a *Minimum access role* for users accessing {product-title} by using the selected identity provider.
++
+[TIP]
+====
+Set the *Minimum access role* to *Admin* while you complete setup.
```

Comment:
This is really a poor tip @gaurav-nelson we'd recommend "None" but admin makes sense only from a one time testing perspective.